protobuf/transform.go

```diff
@@ -130,13 +130,13 @@ func (t *Transformer) transformTypeList(pkg *Package, types []scanner.Type, name
 		return NewGeneratedNamed(toProtobufPkg(pkg.Path), msgName)
 	}
 
-	return t.transformType(pkg, types[0])
+	return t.transformType(pkg, types[0], &Message{}, &Field{})
```

Comment:
but then you have the problem of "i'm changing X in the decorator but that is not actually true because the X was fake"